providers/tentacle.rb

```diff
@@ -172,6 +172,16 @@
     end
     new_resource.updated_by_last_action(actions_updated?([register_polling_instance]))
   else
+
+    firewall = windows_firewall_rule 'Octopus Deploy Tentacle' do
+      localport port.to_s
+      dir :in
+      protocol 'TCP'
+      firewall_action :allow
+      action :create
```

Comment:
Oh I just meant the action attribute not the resource itself 😄  but that's fine.